sqlstore/src/main/resources/db/migration/V0030__project_shortName.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Project ADD COLUMN shortName varchar(5) DEFAULT NULL;
```

Comment:
Okay. I think the existing code would handle this, with the getter doing the substitution.